calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+| ---------------------------------------- | -------- | ------------- | --------------------------------------------------------------------------------------------------------------- |
+| `base_lidar_crop_box_min_x`              | `double` | `-20.0`       | Minimum x-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_crop_box_min_y`              | `double` | `-20.0`       | Minimum y-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_crop_box_min_z`              | `double` | `-20.0`       | Minimum z-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_crop_box_max_x`              | `double` | `20.0`        | Maximum x-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_crop_box_max_y`              | `double` | `20.0`        | Maximum y-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_crop_box_max_z`              | `double` | `20.0`        | Maximum z-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_max_inlier_distance`         | `double` | `0.01`        | Maximum inlier distance in meters for ground extraction by using `pcl::SACSegmentation`.                        |
+| `base_lidar_max_iterations`              | `int`    | `1000`        | The maximum number of iterations for ground extraction by using `pcl::SACSegmentation`.                         |
+| `base_lidar_min_plane_points`            | `int`    | `1000`        | The minimum number of points required in a pointcloud, ensuring sufficient data for applying ground extraction. |
+| `base_lidar_min_plane_points_percentage` | `double` | `10.0`        | The minimum percentage of the ground plane points in a pointcloud.                                              |
+| `base_lidar_max_cos_distance`            | `double` | `0.5`         | Maximum cosine distance for applying ground plane extraction.                                                   |
+| `base_lidar_overwrite_xy_yaw`            | `bool`   | `false`       | Flag to allow overwriting the x, y, and yaw value during base-lidar calibration.                                |
+
+## Known issues/limitations
+
+- As described in [Step 2](#step-2-prepare-calibration-data), the calibrator interpolates the pose of the `mapping lidar` at the timestamp of the `calibration lidars`. Therefore, the calibrator highly requires time synchronization between all of the lidar sensors.
+- It is required that the `mapping lidar` have high resolution. For the `calibration lidar`, both high and low resolutions are acceptable. Therefore, the vehicle with only low-resolution lidars could not use the calibrator.
```

Comment:
grammar

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/cfe975ef9d0f79edca1a55729c0c29a978289eb4

Reply:
Resolution would not be a correct or easy to understand term here.
Probably when you say resolution, at least for mechanical lidars, you mean the number of lines.
Actually, the azimuth resolution is not that critical compared with the number of lines at all.

For mapping in general, the sensor range may be even more important than the number of lines (which of those is more important would depend on the particular scenario though)

Reply:
highly requires is not correct